lib/opensusebasetest.pm

```diff
@@ -417,8 +417,9 @@ sub export_logs {
     $self->save_and_upload_log('systemctl status',          '/tmp/systemctl_status.log');
     $self->save_and_upload_log('systemctl',                 '/tmp/systemctl.log', {screenshot => 1});
 
-    script_run "save_y2logs /tmp/y2logs_clone.tar.bz2";
-    upload_logs "/tmp/y2logs_clone.tar.bz2";
+    my $compression = is_sle('=12-sp1') ? 'bz2' : 'xz';
```

Comment:
We could reuse `get_available_compression` method from:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/y2_logs_helper.pm#L117

Reply:
xz is default compression, I would prefer to use xz.

Reply:
alright